database/migrations/2022_03_11_104113_update_order_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateOrderTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('orders', function (Blueprint $table) {
+            $table->integer('invoice_id')->nullable();
+        });
```

Comment:
this can't be here, it's not related to `fakturowania` it should be `invoices` related

Reply:
done